tools/pipelines/build-api-markdown-documenter.yml

```diff
@@ -42,6 +42,7 @@ trigger:
     - tools/pipelines/templates/include-publish-npm-package-steps.yml
     - tools/pipelines/templates/include-use-node-version.yml
     - tools/pipelines/templates/include-log-test-failures.yml
+    - scripts/report-parser.js    
```

Comment:
This reminds me that there are several other scripts that should be on this list. Most of them are used in some way in the pipeline - maybe just add `scripts/*`?

Reply:
done